terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/main/java/org/terasoluna/gfw/common/date/DateConvertUtils.java

```diff
@@ -93,4 +92,15 @@ public static Time convertToTime(java.util.Date date) {
         Time time = new Time(cal.getTimeInMillis());
         return time;
     }
+
+    /**
+     * Check java.util.Date is not null<br>
+     * @param date check target
+     * @throws IllegalArgumentException if the date is null
+     */
+    private static void CheckDate(java.util.Date date) {
```

Comment:
- [ ] CheckDate -> checkDate

And I think more better to name etc,,, `assertArgs`, `assertDateToConvert`.

Reply:
adopted `assertArgs` and fix javadoc.

262a4b43149e693746834633831e558fa2a25509